src/openforms/js/components/admin/form_design/logic/actions/Action.js

```diff
@@ -58,6 +64,7 @@ const Action = ({prefixText, action, onChange, onDelete}) => {
 Action.propTypes = {
     prefixText: PropTypes.node.isRequired,
     action: ActionType.isRequired,
+    errors: PropTypes.object,
```

Comment:
can we describe this in more detail with `PropTypes.shape` or `PropTypes.objectOf` instead of just `object`? I have a heard time figuring out the datastructure from the code itself